bin/tbg

```diff
@@ -66,9 +66,15 @@ function resolve_vars
 
         var=$(echo -e "$data_set" | cut -d"=" -f1)
         var_value=$(echo -e "$data_set" | cut -d"=" -f2-)
-        # the right hand side is fully resolved so we can evaluate the right hand side
+
         var_value=$(eval echo "$var_value")
-        export "$var=$var_value"
+        if echo "$data_set" | grep -q -e '^.*$(' -e  '^.*${' ; then
```

Comment:
This was the only solution I found to fix the issue. I check for expressions and only evaluate the right hand side if needed to avoid that escaped strings get resolved.